plugin.rb

```diff
@@ -345,6 +345,16 @@ class ::TopicListItemSerializer
     def has_accepted_answer
       object.custom_fields["accepted_answer_post_id"] ? true : false
     end
+
+    if SiteSetting.empty_box_on_unsolved
```

Comment:
this means a site setting change will require a full application reload :( 

just use `include_can_have_answer?`